repondeur/tests/testapp.py

```diff
@@ -12,6 +12,13 @@ class TestResponse(BaseTestResponse):
     def parser(self):
         return HTMLParser(self.text)
 
+    @property
+    def title(self) -> str:
+        return self.parser.css_first("h1").text()
```

Comment:
Nice! When reading the tests I thought it was the contents of the <title> tag. Should we call it first_h1 or something to avoid the ambiguity?